bin/knapsack_pro_rspec

```diff
@@ -3,6 +3,7 @@ if [ "$KNAPSACK_PRO_TEST_SUITE_TOKEN_RSPEC" = "" ]; then
   KNAPSACK_PRO_ENDPOINT=https://api-disabled-for-fork.knapsackpro.com \
     KNAPSACK_PRO_MAX_REQUEST_RETRIES=0 \
     KNAPSACK_PRO_TEST_SUITE_TOKEN_RSPEC=disabled-for-fork \
+    RUBYOPT="-W:deprecated"\
```

Comment:
Makes it easier for me to work towards cleaning up all deprecation warning. It also, weirdly, added back syntax colors in Travis log :smiley:. [Take a look](https://travis-ci.com/github/forem/forem/jobs/495032311#L952)